src/features/identityCheck/components/QuitIdentityCheckModal.tsx

```diff
@@ -25,14 +25,19 @@ export const QuitIdentityCheckModal: FunctionComponent<Props> = ({
   resume,
   testIdSuffix,
 }) => {
+  function quitIdentityCheck() {
+    resume()
```

Comment:
```suggestion
    hideModal()
```

et changer la props `resume` => `hideModal`